File: app/src/main/java/de/markhaehnel/rbtv/rocketbeanstv/objects/schedule/Schedule.java
package de.markhaehnel.rbtv.rocketbeanstv.objects.schedule;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

import lombok.Getter;

@Getter
public class Schedule {

    @SerializedName("schedule")
    @Expose
    private List<ScheduleItem> schedule = new ArrayList<ScheduleItem>();
}

File: app/src/main/java/de/markhaehnel/rbtv/rocketbeanstv/objects/schedule/Schedule.kt
package de.markhaehnel.rbtv.rocketbeanstv.objects.schedule

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import java.util.ArrayList


class Schedule {
    @SerializedName("schedule")
    @Expose
    public val schedule = ArrayList<ScheduleItem>()
}
